acpdr/swagger-specs/audit-query.yaml

```diff
@@ -0,0 +1,332 @@
+swagger: '2.0'
+info:
+  description: >-
+    - Service description:
+        - Adobe Experience Platform allows you to audit user activity for various services and capabilities in the form of audit logs. Each action recorded in a log contains metadata that indicates the action type, date and time, the email ID of the user who performed the action, and additional attributes relevant to the action type.
+        - The Audit Query API allows you to programmatically view and export audit logs recorded by the system.
+
+    - API paths:
+        - PLATFORM Gateway URL: https://<span>platform.adobe.io/
+        - Base path for this API: /data/foundation
+        - Example of a complete path: https://<span>platform.adobe.io/data/foundation/audit/events
+
+    - Required headers:
+        - All calls require the headers `Authorization`, `x-gw-ims-org-id`, and `x-api-key`. For more information on how to obtain these values, see the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+        - All resources in Experience Platform are isolated to specific virtual sandboxes. All requests to Platform APIs require the header `x-sandbox-name` whose value is the all-lowercase name of the sandbox the operation will take place in (for example, "prod"). See the [sandboxes overview](https://adobe.com/go/sandbox-overview-en) for more information.
+  version: '1.0'
+  title: Audit Query API
+host: platform.adobe.io
+basePath: /data/foundation
+tags:
+- name: Audit events
+  description: Audit events are timestamped records of observed activities in Platform. The API allows you to query events over the last 90 days and create export requests.
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  "/audit/events":
+    get:
+      tags:
+      - Audit events
+      summary: Retrieve a list of audit events.
+      operationId: getAuditLogsUsingGET
```

Comment:
Wouldn't this just be getAuditLogs? It feels a little weird to have it as get using GET, since get is the action already.